components/calendar/src/astronomy.rs

```diff
@@ -0,0 +1,657 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+/// This file contains important structs and functions relating to location,
```

Comment:
Module-level Rust doc like this should be `//!` since it describes the module/file.

Another alternative is to still use `///` but put these comments on the `pub mod astronomy` declaration in `lib.rs`, since it's still attached to the same logical construct.